assets/scripts/main.js

```diff
@@ -3,47 +3,32 @@
  *
  */
 
-// Initalization scripts
-import './app/initialization'
+import $ from 'jquery'
+window.$ = $
 
-// Export modules to window for the global-scoped JavaScript
-import './exports'
+import _ from 'lodash'
+window._ = _
 
-// Load things that are needed
-import './app/event_tracking'
+import { Stmx } from './app/initialization'
+window.Stmx = Stmx
+
+// import modules for side-effects
 import './app/blocking_shield'
 import './app/debug_info'
 import './app/keyboard_commands'
 import './app/print'
 import './app/status_message'
 import './app/welcome'
-
-// Menus
-import './menus/menu'
-import './menus/_feedback'
-import './menus/_help'
-import './menus/_identity'
-import './menus/_settings'
-import './menus/_share'
-
-// Dialogs
-import './dialogs/_about'
-import './dialogs/_save_as_image'
```

Comment:
The `/dialogs/_save_as_image` actually does have a side effect in that it attaches an event listener to the "Save as image..." menu item so that it opens a dialog. See here on staging: https://streetmix-staging.herokuapp.com/ -- click on Share -> Save as image... Nothing happens, and you'll see the URL append a `#` which is what happens when the event didn't fire.

What polyfills do you mean?